packages/stream_chat_flutter/lib/src/video/vlc/vlc_manager_desktop.dart

```diff
@@ -1,11 +1,11 @@
-import 'package:dart_vlc/dart_vlc.dart';
+import 'package:media_kit/media_kit.dart';
 import 'package:stream_chat_flutter/src/video/vlc/vlc_manager.dart';
 
 /// The desktop implementation of [VlcManager]. It simply initializes VLC.
 class VlcManagerDesktop extends VlcManager {
   @override
   void initialize() {
-    DartVLC.initialize();
+    MediaKit.ensureInitialized();
```

Comment:
Any reason for switching to MediaKit from VLC?

Reply:
(Just personal curiosity)

Reply:
Mainly because DartVLC has an outdated package dependency which prevent its compatibility with dart 3.22 (and also 3.19).
On the other hand, DartVLC is marked as deprecated and replaced with MesiaKit on pub.dev